enarx-keep-sgx-shim/src/elf.rs

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: Apache-2.0
+
+pub const MAGIC: [u8; 4] = [0x7f, 0x45, 0x4c, 0x46];
+//pub const CLASS_64: u8 = 2;
+//pub const DATA_LE: u8 = 1;
+//pub const VERSION: u8 = 1;
+//pub const OSABI_LINUX: u8 = 3;
+//pub const TYPE_DYN: u16 = 3;
+//pub const MACHINE_AMD64: u16 = 0x3e;
```

Comment:
Should we remove these?

Reply:
My preference would be to remove all of it and depend on a real elf library.

Which brings up one question we need to answer: goblin or xmas-elf? @connorkuehl @haraldh

Reply:
Open an issue for that we investigate which is better.

Reply:
https://github.com/enarx/enarx/issues/425

I will remove the commented stuff for now.